Adding additional docs on sensors and ROS 2 usage (#433)
gazebosim/docs: fortress/sensors.md

an IMU sensor, a Contact sensor and a Lidar sensor. We will also
learn how to launch multiple tasks with just one file using `ign_launch`.

> You can find the final world of this tutorial showing all these plugins in use [here.](https://github.com/gazebosim/docs/blob/master/ionic/tutorials/sensors/sensor_tutorial.sdf)

Author: My intent here is to early on point to helpful resources that folks might only be on this page to obtain (examples, tutorial source, and how to put it together for sensors)

> You can find the final world of this tutorial showing all these plugins in use [here.](https://github.com/gazebosim/docs/blob/master/ionic/tutorials/sensors/sensor_tutorial.sdf)

Reviewer: ```suggestion
You can find the final world of this tutorial showing all these plugins in use [here.](https://github.com/gazebosim/docs/blob/master/fortress/tutorials/sensors/sensor_tutorial.sdf)
```

> You can find the final world of this tutorial showing all these plugins in use [here.](https://github.com/gazebosim/docs/blob/master/ionic/tutorials/sensors/sensor_tutorial.sdf)

Reviewer: Nit
```suggestion
You can find the final world of this tutorial showing all these plugins in use [here](https://github.com/gazebosim/docs/blob/master/ionic/tutorials/sensors/sensor_tutorial.sdf).
```

> You may also find an extensive set of world examples with many possible sensors and actuation capabilities in [`gz-sim/examples/worlds`](https://github.com/gazebosim/gz-sim/tree/gz-sim8/examples/worlds) for individual examples.

Reviewer: Since this doc is for Fortress, it should link to ign-gazebo6 branch
```suggestion
You may also find an extensive set of world examples with many possible sensors and actuation capabilities in [`gz-sim/examples/worlds`](https://github.com/gazebosim/gz-sim/tree/ign-gazebo6/examples/worlds) for individual examples.
```

> The full set of sensors can be found in the [`gz-sensors` project](https://github.com/gazebosim/gz-sensors).

Reviewer: ```suggestion
The full set of sensors can be found in the [`gz-sensors` library](https://github.com/gazebosim/gz-sensors).
```

If using ROS, you can see demo launches and bridging configuration for these examples [here](https://github.com/gazebosim/ros_gz/tree/ros2/ros_gz_sim_demos/launch).

## Preliminaries

> When adding a `plugin` to an SDF file which does not currently contain one, the default plugins are not loaded. Before adding a sensor, make sure to add in a couple of logical defaults to your world so that it is possible to continue to use the GZ GUI:

Author: This is a real issue I ran into that blocked me for several hours requiring a conversation with @azeey since there were no errors. I used the SDF exported by the GUI exporter which did not contain the plugins that were used by the GUI (which seems like a bug; shouldn't that export the SDF being full complete?). Thus, when I added sensor plugins, the SDF didn't load in the GUI anymore. This was due to missing plugins that this will help make sure others don't run into in the future while following this tutorial from their given SDF files.

Reviewer: Sounds reasonable. I'm not an expert in this. But this sounds like something I might just assume and copy-paste whereas a non-developer might not know. So it seems a good addition to me.
